src/utils/browser.ts

```diff
@@ -0,0 +1,15 @@
+export const recommendedBrowsers: string[] = [
+    "Chrome",
+    "Firefox"
+  ]
+  
+  export function isRecommendedBrowser() {
+    for (let i = 0; i < recommendedBrowsers.length; i++) {
```

Comment:
I think that changing this for
```
return recommendedBrowsers.some((browser) => navigator.userAgent.indexOf(browser) !== -1)
```
Might be cleaner. What do you think?

Reply:
I like it! Changed in https://github.com/decentraland/explorer-website/pull/34/commits/a3ce9a141f04a8070ee3adbbd9f3e6c502e3c495. Thanks!